.github/workflows/spell_checking.yml

```diff
@@ -0,0 +1,33 @@
+name: Spell Checking
+
+on: [pull_request]
+
+jobs:
+  codespell:
+    name: Check spelling of all files with codespell
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install codespell
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Check spelling with codespell
+        run: codespell --ignore-words=codespell.txt || exit 1
+  misspell:
+    name: Check spelling of all files in commit with misspell
```

Comment:
Won't `git ls-files` list all repo files, not just files in commit?

Reply:
There's one file in the repo, so we might just as well point to it. :-)

Reply:
I'm sorry about I seem to lack understanding of the issue. What kind of improvement can be made to that?

Reply:
I was thinking that "Check spelling with misspell" is sufficient.